test/os/ui/draw/draw.test.js

```diff
@@ -0,0 +1,67 @@
+goog.require('ol.Feature');
+goog.require('ol.geom.Polygon');
+goog.require('ol.proj');
+goog.require('os.map');
+goog.require('os.ui.draw');
+
+describe('os.ui.draw', function() {
+  it('should read grid settings as numbers', function() {
```

Comment:
This test is superfluous. You've typed `osx.ui.draw.GridOptions` already, the compiler will handle it.

Reply:
Cool cool -- removed.